source/leuven/services/openondemand.rst

```diff
@@ -191,136 +243,333 @@ of using any of the other partitions. There are however some limitations on the
-packages you need):
+There are some package requirements if you want to use R in code-server.
+The following command creates a functional environment (of course, add any other packages you need):
 
         .. code-block:: bash
 
          conda create -n <env_name> -c conda-forge r-base r-remotes r-languageserver r-httpgd r-jsonlite
 
-Once you've created your environment, go ahead and start a code-server session on Open Ondemand. On the lefthand side, go to the extension menu and search
-for 'R'. You should install the 'R' extension of 'REditorSupport'.
+Once you've created your environment, go ahead and start a code-server session on Open Ondemand.
+On the lefthand side, go to the extension menu and search for 'R'.
+You should install the 'R' extension of 'REditorSupport'.
+
+Now there are two ways to use the R installation inside your Conda environment:
+
+- Open a terminal (three horizontal lines in the upper left corner - Terminal - New Terminal),
+  and activate your Conda environment.
+  Now type ``R`` in the terminal and you will be able to use your scripts interactively
+  (R gets attached as soon as you start it).
+- You can also set the path to the R version that needs to be attached (better if you always
+  use the same Conda environment).
+  Go to 'Extensions', and click the settings wheel next to the R extension.
+  Select 'Extension Settings' and search for the 'R > RTerm: Linux' setting.
+  Paste the path to your Conda env there (``/path/to/miniconda/envs/<env_name>/lib/R``)
+
+**Remarks:**
+
+- To run your script line-by-line, place your cursor on a desired line, and press the key combination of
+  'ctrl+enter' on your keyboard.
 
-Now there are two ways to use the R installation inside your conda environment:
 
-- Open a terminal (three horizontal lines in the upper left corner - Terminal - New Terminal), and activate your conda environment. Now type 'R' in the terminal and you will be able to use your scripts interactively (R gets attached as soon as you start it).
-- You can also set the path to the R version that needs to be attached (better if you always use the same conda environment). Go to 'Extensions', and click the settings wheel next to the R extension. Select 'Extension Settings' and search for the 'R > RTerm: Linux' setting. Paste the path to your conda env there (`/path/to/miniconda/envs/<env_name>/lib/R`)
+.. _ood_matlab_app:
+
+MATLAB
+------
+
+To launch MATLAB via OnDemand, you need to additionally specify your desired version of the software
+from the drop-down menu on the resource form.
+Given that our current MATLAB installations automatically detect GPU devices and CUDA libraries,
+you may also request GPU(s) as resources, if needed.
+
+Once you launch the session, a remote `noVNC`_ desktop will start on a compute node.
+Once the session starts, the selected MATLAB module will be loaded, and eventually the MATLAB GUI
+will pop up (after waiting for few seconds).
+
```

Comment:
Best to add here an additional remark, that Matlab will only be usable by users that are in a license group for Matlab.
This remark can be site specific, as for us this is group lli_matlab but could be something else for Gent, Antwerp or VUB.

Reply:
Given that this page is already site-specific, I just added a 'Note' block about the license. Please see 1e67e1a.